### ISSUE TITLE
Need an addition to the README to explain how to use with docker

### ISSUE THREAD

**Issue by richtong** [bclinkinbeard/egghead-hjs-webpack-demo#2]
I pulled my hair out trying to figure out why a simple dockerization doesn't work. Turns out that there are two changes needed to make it work. First is to add `--inline` to the package.json call to the webpack-dev-server so the request `http://localhost:3000/` works and if you do not want to do that then with the current code, you must use the iframe mode of docker and you can see the application working properly at `http://localhost:3000/webpack-dev-server/index.html`.

See http://stackoverflow.com/questions/33462123/connecting-webpack-dev-server-inside-a-docker-container-from-the-host as the only post that explains this. The fix seems pretty simple and I forked your repo and did this in https://github.com/surround-io/egghead-hjs-webpack-demo

**Comment by bclinkinbeard** [comment]
Hey Rich,

Thanks for the heads up. Did you forget to push your changes? I don't see any new commits in your fork.

**Comment by richtong** [comment]
Oh I just left u with a comment. I realized my fork went way farther. Added a Makefile to generate docker containers and debug them. As well as the fix I told u about. Also added a README. Do u want these. Most of these react tutorials seem to only use native machines and our shop is completely dockerized. 

Thanks,
Rich

No! Try not. Do, or do not. There is no try. Master Yoda. Star Wars Episode V: The Empire Strikes Back

http://tongfamily.com | @richtong | http://fb.me/richtong1 | +1-206-963-9029
